README.md

```diff
@@ -73,7 +73,8 @@ The report shows up at `coverage/html/index.html` viewable in a browser.
 
 OpenSeadragon is truly a community project; we welcome your involvement!
 
-When contributing, please attempt to match the code style already in the codebase. Note that we use four spaces per indentation stop. For more thoughts on code style, see https://github.com/rwldrn/idiomatic.js/.
+When contributing, please attempt to match the code style already in the codebase. Note that we use four spaces per indentation stop. For easier setup you can also install [EditorConfig](http://editorconfig.org/) if your IDE is supported. For more thoughts on code style, see ®
```

Comment:
I'm seeing a registered trademark symbol at the end of this paragraph...can you remove that? Also, presumably the idiomatic link below should be part of this line rather than having a newline before it.